sirepo/raydata/analysis_driver/__init__.py

```diff
@@ -75,6 +75,8 @@ def get_papermill_args(self):
         ]:
             res.extend(["-p", f"'{n}'", f"'{v}'"])
         res.append("--report-mode")
+        res.append("--log-output")
```

Comment:
These make the output way better thank you:)

How about wrapping them up in a single call? I think it is a bit cleaner.

Like
```python
res.extend(('--report-mode', '--log-output', '--no-progress-bar'))
```

Reply:
49fa5cc88665cf01d9f4cd3fb82e446484e0f1bd